[2024-09-09T00:00:00.743+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-09T00:00:00.769+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.first_task scheduled__2024-09-08T00:00:00+00:00 [queued]>
[2024-09-09T00:00:00.792+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.first_task scheduled__2024-09-08T00:00:00+00:00 [queued]>
[2024-09-09T00:00:00.796+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-09T00:00:00.812+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): first_task> on 2024-09-08 00:00:00+00:00
[2024-09-09T00:00:00.831+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=20538) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-09T00:00:00.830+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'first_task', 'scheduled__2024-09-08T00:00:00+00:00', '--job-id', '290', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpzssdi26g']
[2024-09-09T00:00:00.837+0000] {standard_task_runner.py:105} INFO - Job 290: Subtask first_task
[2024-09-09T00:00:00.836+0000] {standard_task_runner.py:72} INFO - Started process 20550 to run task
[2024-09-09T00:00:00.962+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v3.first_task scheduled__2024-09-08T00:00:00+00:00 [running]> on host 626f2e28dec2
[2024-09-09T00:00:01.104+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-08T00:00:00+00:00'
[2024-09-09T00:00:01.111+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-09T00:00:01.137+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-09T00:00:01.144+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-09T00:00:01.176+0000] {subprocess.py:86} INFO - Output:
[2024-09-09T00:00:01.189+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-09T00:00:01.191+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-09T00:00:01.218+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-09T00:00:01.219+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=first_task, run_id=scheduled__2024-09-08T00:00:00+00:00, execution_date=20240908T000000, start_date=20240909T000000, end_date=20240909T000001
[2024-09-09T00:00:01.330+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-09T00:00:01.403+0000] {taskinstance.py:3891} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-09-09T00:00:01.407+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
